editor/src/components/navigator/navigator-item/component-picker.tsx

```diff
@@ -190,6 +238,7 @@ const ComponentPickerTopSection = React.memo((props: ComponentPickerTopSectionPr
         flexDirection: 'column',
       }}
     >
+      <FilterButtons components={components} />
```

Comment:
🚢  If someone wants to ship the design changes without me, I think you just need to comment this out.